backend/Alpaki.WebApi/Startup.cs

```diff
@@ -63,30 +66,26 @@ public void ConfigureServices(IServiceCollection services)
 
             string privateSecretKey = Configuration.GetValue<string>("SeacretKey");
 
-            services.AddAuthentication(options =>
-            {
-                options.DefaultAuthenticateScheme = JwtBearerDefaults.AuthenticationScheme;
-                options.DefaultChallengeScheme = JwtBearerDefaults.AuthenticationScheme;
-            })
+            services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
             .AddJwtBearer(options =>
             {
-                options.RequireHttpsMetadata = false;
-                options.SaveToken = true;
                 options.TokenValidationParameters = new TokenValidationParameters
                 {
-                   
-                    ValidateIssuerSigningKey = true,
-                    IssuerSigningKey = new SymmetricSecurityKey(FromBase64Url(privateSecretKey)),
                     ValidateIssuer = false,
-                    ValidateAudience = false
+                    ValidateAudience = false,
+                    ValidateLifetime = false,
+                    ValidateIssuerSigningKey = true,
+                    IssuerSigningKey = new SymmetricSecurityKey(Encoding.UTF8.GetBytes(privateSecretKey))
                 };
             });
 
             RegisterGraphQL(services);
 
-            services.RegisterLogicServices(privateSecretKey);
+            services.RegisterLogicServices();
 
             services.AddControllers();
+            services.AddHttpContextAccessor();
+            services.AddMediatR(typeof(InitializeLogic).GetTypeInfo().Assembly);
```

Comment:
Jest już to odpalone w RegisterLogicServices

Reply:
hmmm wygląda jak merge conflikt